hoprd/hoprd/src/lib.rs

```diff
@@ -135,10 +136,13 @@ impl hopr_lib::HoprSessionReactor for HoprServerIpForwardingReactor {
                     "UDP target {udp_target} resolved to {resolved_udp_target}"
                 );
 
-                let udp_bridge = hopr_network_types::udp::ConnectedUdpStream::bind(("0.0.0.0", 0))
-                    .await
-                    .and_then(|s| s.with_counterparty(resolved_udp_target))
-                    .map(|s| s.with_foreign_data_mode(ForeignDataMode::Error))
+                let mut udp_bridge = hopr_network_types::udp::ConnectedUdpStream::builder()
+                    .with_buffer_size(HOPR_UDP_BUFFER_SIZE)
+                    .with_counterparty(resolved_udp_target)
+                    .with_foreign_data_mode(ForeignDataMode::Error)
+                    .with_queue_size(2048)
```

Comment:
Isn't it too small?
```suggestion
                    .with_queue_size(2048)
```

Reply:
This is 2048 * `HOPR_UDP_BUFFER_SIZE` which is 16k, therefore the queue can store `32M` per Session. Maybe for nodes where we know there will be only a handful of Sessions, we can set this higher.